jcl/src/java.base/share/classes/java/lang/Throwable.java

```diff
@@ -319,12 +314,29 @@ public void printStackTrace (PrintStream err) {
  *				The writer to write the walkback on.
  */
 public void printStackTrace(PrintWriter err) {
+	printStackTraceHelper(err);
+}
+
+/**
+ * Outputs representation of the receiver's
+ * walkback on the Appendable specified by the argument.
+ *
+ * @param	appendable Appendable 
+ *		The Appendable object to the walkback will be written.
+ */
+private void printStackTraceHelper(Appendable appendable) {
 	StackTraceElement[] stack;
-	stack = printStackTrace(err, null, 0, false);
+	Set<Throwable> exceptionChainSet = null;
+	try {
+		exceptionChainSet = Collections.newSetFromMap(new IdentityHashMap<Throwable, Boolean>());
+	} catch (OutOfMemoryError e) {
+		exceptionChainSet = null;
```

Comment:
This is redundant, the variable is already initialized to null. A comment is sufficient.

Reply:
Done.